streams/test-utils/src/main/java/org/apache/kafka/streams/test/ConsumerRecordFactory.java

```diff
@@ -215,7 +236,26 @@ public void advanceTimeMs(final long advanceMs) {
                                                  final V value) {
         final long timestamp = timeMs;
         timeMs += advanceMs;
-        return create(topicName, key, value, timestamp);
+        return create(topicName, key, value, new RecordHeaders(), timestamp);
+    }
+
+    /**
+     * Create a {@link ConsumerRecord} with the given topic name, key, headers, and value.
+     * The timestamp will be generated from the constructor provided and time will auto advance.
```

Comment:
Seems to be c&p error: `The timestamp will be generated [based on] the constructor provided [start time] and time will auto advance.`

Can you fix everywhere?

Reply:
ack.